perma_web/frontend/lib/store.js

```diff
@@ -12,14 +13,57 @@ export const getLinksRemainingStatus = (linksRemaining, isNonpaying) => {
     return globalStore.updateLinksRemainingStatus('unlimited_paid')
 }
 
-export const getUserTypes = (isIndividual) => {
-    let types = []
-
+export const getUserTypes = (isIndividual, user) => {
     if (isIndividual) {
-        types = types.concat('individual')
+        return globalStore.updateUserTypes('individual')
+    }
+
+    let userTypes = []
+
+    const topLevelFolders = user.top_level_folders
+    const isOrgAffiliated = topLevelFolders.some(folder => !!folder.organization)
```

Comment:
Let's pass this info [straight to the javascript from the Django app](https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/templates/user_management/create-link.html#L163-L173), rather than having to calculate it by iterating through all the user's folders. I think that will be much less expensive, especially for admins, who have lots and lots of top level folders :-)

Here's the current method for calculating `is_individual`: you'll see it actually checks all the varieties: https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/models.py#L997... 

We could gather this info efficiently, without too many round trips back and forth with the database, but for now, how about just passing the info straight to the JS without worrying about that? We can clean up afterwards.

Reply:
For sure! Great callout — refactor this in 9b238bb1bc24b2404c7950532c6837ce224fd403 to pull this information directly from the user response object. 

I also pulled in `is_staff` in [5b61873](https://github.com/harvard-lil/perma/pull/3507/commits/5b618738af2477d0630640656ac32490a59d77eb) because I noticed that admin users didn't have a folder select that was populating with all organization data.

Reply:
👍 neato.

question: have you checked to see how this works for registrar users? only asking to make sure, since the attribute in JS is now `isOrganizationUser` rather than `isOrgAffiliated`. Do you need to explicitly account for registrar users now too?

Reply:
Ah, no — I was testing with a registrar user I'd previously also marked as staff. This should now be handled in [183aff7](https://github.com/harvard-lil/perma/pull/3507/commits/183aff736da6741ded954b0ffa047b79ac133f5b)